README.md

```diff
@@ -49,41 +49,56 @@ $ sha256sum -c --strict SHA256SUM.md
 
 ## Verification of Provenance
 
+We currently support artifact verification (for binary blobs) and container images.
+
 ### Available options
 
-Below is a list of options currently supported. Note that signature verification is handled seamlessly without the need for developers to manipulate public keys.
+Below is a list of options currently supported for binary blobs and container images. Note that signature verification is handled seamlessly without the need for developers to manipulate public keys. See [Available options](#available-options) for details on the options exposed to validate the provenance.
 
 ```bash
 $ git clone git@github.com:slsa-framework/slsa-verifier.git
-$ go run ./cli/slsa-verifier --help
- Usage of ./slsa-verifier:
-  -artifact-path string
-    	path to an artifact to verify
-  -branch string
-    	expected branch the binary was compiled from (default "main")
-  -print-provenance
-    	output the verified provenance
-  -provenance string
-    	path to a provenance file
-  -source string
-    	expected source repository that should have produced the binary, e.g. github.com/some/repo
-  -tag string
-    	[optional] expected tag the binary was compiled from
-  -versioned-tag string
-    	[optional] expected version the binary was compiled from. Uses semantic version to match the tag
+$ go run ./cli/slsa-verifier/ verify-artifact --help
+Verifies SLSA provenance on an artifact blob
+
+Usage:
+  slsa-verifier verify-artifact [flags]
+
+Flags:
+      --branch string            [optional] expected branch the binary was compiled from
+      --builder-id string        EXPERIMENTAL: the unique builder ID who created the provenance
+  -h, --help                     help for verify-artifact
+      --print-provenance         print the verified provenance to std out
+      --provenance-path string   path to a provenance file
+      --source string            expected source repository that should have produced the binary, e.g. github.com/some/repo
+      --tag string               [optional] expected tag the binary was compiled from
+      --versioned-tag string     [optional] expected version the binary was compiled from. Uses semantic version to match the tag
+      --workflow-input map[]     [optional] a workflow input provided by a user at trigger time in the format 'key=value'. (Only for 'workflow_dispatch' events). (default map[])
```

Comment:
this would (maybe ?) be renamed to `build-inputs` if we follow the `build` / `source` / `provenance` requirements's naming convention. I'm not 100% sure `build-input` is better than `workflow-input`... maybe `build-workflow-input`...

Reply:
I think `build-inputs` is an accurate term for workflow dispatch inputs, but i wonder if that would get conflated with other types of inputs.

For GHA builders: the only inputs to workflows is through workflow dispatch inputs. But what about inputs to the trusted builders themselves?

Reply:
full agreed. That's why I hinted at `-build-workflow-inputs`., to make it more explicit that it's for (GitHub) workflows. Not sure this is enough. Is it possible to name the option after the type of builders that support it? `-build-gha-inputs`. I'm making the name up...

Reply:
I think formatting inputs is brittle and if we are considering that it is likely not good. If they are builder specific, I think some options are already interpreted slightly different -- so `build-input` for GHA could mean workflow inputs. 

OTOH I don't know how to distinguish between types of build inputs for a single builder. If someone wants to check on inputs to the trusted builders, can they access the provenance, since that is more of a "recipe"?

Reply:
For now, I think it's easier to organize as `build-XYZ` or `source-XYZ` of the current options. At least it organizes the options.

Reply:
SGTM. 

> OTOH I don't know how to distinguish between types of build inputs for a single builder. If someone wants to check on inputs to the trusted builders, can they access the provenance, since that is more of a "recipe"?

fair enough. Let's start with that then. I am not even sure GCB supports "input", since it's just a webhook.

@ianlewis any objection the direction we're taking here?